pkgs/build-support/setup-hooks/compress-man-pages.sh

```diff
@@ -9,7 +9,7 @@ compressManPages() {
     echo "gzipping man pages under $dir/share/man/"
 
     # Compress all uncompressed manpages.  Don't follow symlinks, etc.
-    find "$dir"/share/man/ -type f -a '!' -regex '.*\.\(bz2\|gz\)$' -print0 \
+    find "$dir"/share/man/ -type f -a '!' -regex '.*\.\(bz2\|gz\|xz\)$' -print0 \
```

Comment:
Shouldn't the `find` below be changed too?

Reply:
yes, d'oh.